src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
```

Comment:
The variable `$gap` is used but not defined or imported in this file. Ensure that it is defined in a variables file or imported correctly.